File: app/src/debug/java/com/github/gfx/helium/StethoDelegator.java
package com.github.gfx.helium;

import com.facebook.stetho.Stetho;
import com.facebook.stetho.okhttp3.StethoInterceptor;

import android.app.Application;

import okhttp3.Interceptor;

public class StethoDelegator {

    public static void setup(Application context) {
        Stetho.initializeWithDefaults(context);
    }

    public static Interceptor createNetworkInterceptor() {
        return new StethoInterceptor();
    }
}

File: app/src/debug/java/com/github/gfx/helium/StethoDelegator.kt
package com.github.gfx.helium

import com.facebook.stetho.Stetho

import android.content.Context

import javax.inject.Inject

class StethoDelegator(application: HeliumApplication) {

    @Inject
    private lateinit var context: Context

    init {
        application.component.inject(this)
    }

    fun setup() {
        Stetho.initializeWithDefaults(context)
    }
}
